report.Rmd

```diff
@@ -786,15 +785,15 @@ In this section, a smaller dataset is created and it is verified if the dropout
 
 Replicating  the drop_general variable to see how many instances are dropped generally when the indicator == 1; this means the individual has dropped from experiment
 
-> table(data$drop_general)
+> table(df$drop_general)
 
    0  |  1 
 1014  |  6 
 
 
 Replicating the drop_on_task variable to see how many instances are dropped on task when the indicator == 1; this means the individual has dropped from experiment
 
-> table(data$drop_on_task)
+> table(df$drop_on_task)
```

Comment:
Maybe use `r` code chunk so it is not hardoced but relies on loaded data?

Reply:
Its better to create a new dataset to create a new table or new figure from existing dataset: For eg:

```{r}
# recode some columns
df <- df %>% 
    mutate(BDG = case_when(game_type == "Bully Dictator Game" ~ "BDG", .default = "DG"),
           BDG = factor(BDG, levels = c("DG", "BDG")),
           treatment_group_factor = factor(x = treatment_group, levels = c("C", "TD", "MD")))
# create models
model_list <- lapply(X = c("ee", "nb", "ne"), 
                     FUN = function(i) fit_tobit(data = df, predictors = c("BDG", "treatment_group_factor"), to_predict = i, limits = list(left = 10, right = 60)))
# print out
lapply(X = model_list, FUN = pretty_t_coefftest) %>% 
    to_frame_multi() %>% 
    mutate(vars = c("BDG", "TD", "MD", "log likelihood")) %>%
    kable(col.names = c(" ", "\\(1) EE", "\\(2) NB", "\\(3) NE"), escape = F) %>% 
    kable_styling()
```

Here its better to create new dataset "df_new" or anything and then create a new table :)